netbox/project-static/src/select/static.ts

```diff
@@ -17,13 +17,18 @@ export function initStaticSelects(): void {
 
 // Initialize color selection fields
 export function initColorSelects(): void {
+  function renderColor(item: TomOption, escape: typeof escape_html) {
```

Comment:
This is really nice!

Reply:
Thanks! I really like how it turned out.